Makefile

```diff
@@ -77,14 +77,34 @@ vet:
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
-# Build the docker image
+# Build a single-architecture docker image
 docker-build: test
 	docker build -t ${IMG} .
```

Comment:
Need to add the default platform in the build command. But I don't think it's needed if we use the hadolint solution mentioned earlier.

```text
Step 1/19 : FROM --platform=${BUILDPLATFORM} golang:1.16 as builder
failed to parse platform : "" is an invalid component of "": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument
```

Reply:
ah sorry – forgot to add `DOCKER_BUILDKIT=1` here to have Docker set `BUILDPLATFORM` by default